cdap-standalone/src/main/java/co/cask/cdap/StandaloneMain.java

```diff
@@ -289,7 +289,12 @@ public static void main(String[] args) {
         usage(false);
         return;
       } else {
-        usage(true);
+         PrintStream out = System.err;
```

Comment:
Also, the variable name ``out`` for ``System.err`` is a little misleading. Should we instead use ``err``, if absolutely required, else stick with ``System.err``, since ``System.err`` is used after this block as well, on line 307.